QueueProcessors/README.md

```diff
@@ -1,9 +1,9 @@
-## QueueProcessors *(Linux Only)*
```

Comment:
Both the Web application and the new EoRM are platform independent services making the QueueProcesor the only service that MUST run on linux based systems due to it's implementation of Daemons. I agree that having this information in the title may be unnecessary. However, as the only people likely to be reading this will be of a technical nature, I think this information should still be captured on this page.